netsim/augment/groups.py

```diff
@@ -128,6 +132,19 @@ def check_group_data_structure(
       continue
 
     gpath=f'{gpath}.{grp}'
+    g_type = gdata.get('type','node')
+    gt_values = ['node','vlan','vrf']
```

Comment:
Why can this not be defined in YAML using ```valid_values```?

Reply:
Because we select the data type to validate against based on g_type in https://github.com/ipspace/netlab/pull/1529/files/060574491a548f534c1c18f76bff102e4098a366#diff-3ef4088ee26c6ab1b27ac441ecf2cb14b48299ee8414cf65f84efccb391e2beeR144